### PR TITLE
Set mtls to STRICT mode when running tests with istio enabled

### PR DESCRIPTION
**Pull request by sankarpn**
Create PeerAuthentication with mtls mode to be STRICT and verify the operator, WebLogic pods are coming up.

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/16333/

### CODE REVIEW

**Review by anpanigr — COMMENTED** [review]
Update the header of file ItIstioMiiDomain.java to use year 2023

**Comment by sankarpn** [comment]
> Update the header of file ItIstioMiiDomain.java to use year 2023

Fixed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java [hunk 1]
```diff
@@ -364,4 +371,18 @@ private DomainResource createDomainResource(String domainUid, String domNamespac
...
+    assertDoesNotThrow(() -> {
+      copyFile(Paths.get(RESOURCE_DIR, "istio", "istio-tls-mode.yaml").toFile(),
+          Paths.get(WORK_DIR, "istio-tls-mode.yaml").toFile());
+    });
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString(),
+          "NAMESPACE", namespace);
+    });
+    ExecResult result = assertDoesNotThrow(() -> ExecCommand.exec(KUBERNETES_CLI + " apply -f "
+        + Paths.get(RESOURCE_DIR, "istio", "istio-tls-mode.yaml").toString(), true));
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java:385]: Do we need to use the file under WORK_DIR instead?
Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString()

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java [hunk 2]
```diff
@@ -364,4 +371,18 @@ private DomainResource createDomainResource(String domainUid, String domNamespac
...
+    assertDoesNotThrow(() -> {
+      copyFile(Paths.get(RESOURCE_DIR, "istio", "istio-tls-mode.yaml").toFile(),
+          Paths.get(WORK_DIR, "istio-tls-mode.yaml").toFile());
+    });
+    assertDoesNotThrow(() -> {
+      replaceStringInFile(Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString(),
+          "NAMESPACE", namespace);
+    });
+    ExecResult result = assertDoesNotThrow(() -> ExecCommand.exec(KUBERNETES_CLI + " apply -f "
+        + Paths.get(WORK_DIR, "istio", "istio-tls-mode.yaml").toString(), true));
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java:385]: remove "istio" parameter here. should be 
Paths.get(WORK_DIR, "istio-tls-mode.yaml").toString()

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by xiancao — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java (hunk 1 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java]: Good catch, fixed.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java (hunk 2 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMiiDomain.java]: Fixed.

**Pull request merged**